src/query/service/src/interpreters/interpreter_table_add_column.rs

```diff
@@ -124,6 +130,30 @@ impl Interpreter for AddTableColumnInterpreter {
             .meta
             .add_column(&field, &self.plan.comment, index)?;
 
+        if let Some(auto_increment) = &self.plan.auto_increment {
+            let mut schema = TableSchema::clone(&table_info.schema()).clone();
+            let index = schema.index_of(field.name.as_str())?;
+            let auto_increment_key =
+                AutoIncrementKey::new(tbl.get_id(), schema.field(index).column_id());
+            let auto_increment_ident =
+                AutoIncrementIdent::new_generic(self.ctx.get_tenant(), auto_increment_key);
+            schema.fields[index].auto_increment_display = Some(auto_increment.to_sql_string());
+
+            table_info.meta.schema = Arc::new(schema);
+
+            let req = CreateSequenceReq {
```

Comment:
Better add some comment why create drop internal seq and add drop column doesn't atomic